src/librbd/crypto/BlockCrypto.cc

```diff
@@ -2,7 +2,9 @@
 // vim: ts=8 sw=2 smarttab
 
 #include "librbd/crypto/BlockCrypto.h"
-#include <alloca.h>
+
+#include <stdlib.h>
```

Comment:
Nit: instead of adding blank lines around this include perhaps just move it down under the include for "ceph_assert". We don't want "byteorder.h" or "ceph_assert" to be able to get things "for free" by including "stdlib" first.

Reply:
@dillaman 
I do not mind doing that.
I was just trying to follow @tchaikov 's pointer in the the coding guide.
(Which wants the empty lines.)